pages/index.tsx

```diff
@@ -1,39 +1,93 @@
-import { Inter } from 'next/font/google'
-import { Box, Button, Image, Flex } from '@chakra-ui/react'
-import { useConnection } from '@/packages/providers'
-import { useRouter } from 'next/router'
+import { Inter } from "next/font/google";
+import {
+  Box,
+  Button,
+  Image,
+  Flex,
+  Text,
+  useBreakpointValue,
+} from "@chakra-ui/react";
+import { useConnection } from "@/packages/providers";
+import { useRouter } from "next/router";
+import Footer from "../components/Footer/Footer";
 
-const inter = Inter({ subsets: ['latin'] })
+const inter = Inter({ subsets: ["latin"] });
 
 export default function Home() {
-  const {address, connect} = useConnection()
-  const router = useRouter()
+  const { address, connect } = useConnection();
+  const router = useRouter();
+  const displayBelow768 = useBreakpointValue({ base: true, md: false });
```

Comment:
Instead of calling `displayBelow768` let's give more meaningful name. Something like, `displayMobile` or `displayTablet`